packages/connectivity/connectivity/example/pubspec.yaml

```diff
@@ -5,9 +5,9 @@ dependencies:
   flutter:
     sdk: flutter
   connectivity:
-    path: ../
+    path: ../../connectivity
   connectivity_macos:
-    path: ../connectivity_macos
+    path: ../../connectivity_macos
```

Comment:
I would depend on the published version of this one (once it is published), so you don't have to pierce outside of your own package to access it.

Reply:
Ah, now that I mention this. You don't need this dependency *at all* once you make connectivity_macos the "endorsed" implementation of the plugin for the macos platform (which you should do for sure :P)

Reply:
I actually can't do that yet, without making some changes to the macos package. I think those changes should be done in a separate PR, I think for now we should leave it in. WDYT?

Reply:
SGTM!